File: src/syntax/lesson14_Scanner/ScannerDemo.java
package syntax.lesson14_Scanner;

import java.util.Scanner;

public class ScannerDemo {
    static void main() {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter your name: ");
        String name = scanner.nextLine().trim();

        System.out.print("Enter your age: ");
        int age = scanner.nextInt();

        System.out.println("Hello, my name is: " + name + " my age is: " + age);

    }
}

File: src/syntax/lesson14_Scanner/ScannerDemo.kt
package syntax.lesson14_Scanner

import java.util.Scanner

fun main() {

    var scanner: Scanner = Scanner(System.`in`)

    print("Enter your name: ")

    val name = scanner.nextLine()

    print("Enter your age: ")

    val age = scanner.nextInt()

    println("My name is $name and I'm $age years old")

    scanner.close()
}